### DISCUSSION TITLE
Traffic Pattern

### DISCUSSION THREAD

**Discussion by Lewis2023** [albar965/littlenavmap#999] [Q&A]
I try to check my traffic pattern flight with LNM with a Cessna172S.
At the point 500ft agl I turn to the left, but in LNM ist the Track only in the half way.
Which parameter is LNM using to calculate the turning points?

**Comment by albar965** [comment]
Not sure if I understand the issue but you can disable `Base 45° after threshold` and change it according to your needs. The tooltips on the input fields give more information about the values.

**Reply by Lewis2023** [reply]
Yes, but I dont understand why the middle of the pattern is centered to the runway. In my opinion the center of the traffic pattern should be at the starting point. I dont start at the center of the runway.

**Reply by albar965** [reply]
So you want to start the crosswind leg right after takeoff still over the runway?
LNM does not know your takeoff distance. This depends on the aircraft.

**Reply by Lewis2023** [reply]
Sorry for my bad explanation. 
I start at the begin of the runway, roughly at the same point I will touch for the landing. . I think this point should be the middle of the pattern (now it is the symbol of the airport). In other words the base leg is further away form the center as the crosswind leg.

**Reply by albar965** [reply]
The center is not the airport symbol but the runway center. The distances are measured from the runway ends. 

So the crosswind leg should start right at the reference point (runway threshold) instead of runway end?

**Reply by Lewis2023** [reply]
Thanks for your answer. I have flown a lot of upwinds with different vertical speeds (angles) to match your pattern line.
With the cessna172 with 74 KIAS I cross the 500ft AGL turn point in the half of the upwind leg distance.
The reason seems now clear to me: LNM cannot calculate this distance because my parameters of the climb are unknown.
(This has nothing to do with the performance file climb rate). Thanks again for this super program

**Reply by albar965** [reply]
Right. It is also unknown if a pilot does only touch an go or if a full takeoff or landing distance has to be considered.
I'd just ignore the crosswind leg and turn whenever it makes sense.
